pom.xml

```
@@ -53,6 +53,7 @@
         <clowder-quarkus-config-source.version>1.0.0</clowder-quarkus-config-source.version>
 
         <cloudwatch.version>4.2.0</cloudwatch.version>
+        <sentry.version>1.1.0</sentry.version>
```

`sentry.version` is a bit misleading. This would help avoiding confusion:
```suggestion
        <quarkus-logging-sentry.version>1.1.0</quarkus-logging-sentry.version>
```

---

This needs to be applied in all pom.xml files now 😃